### PR TITLE
feat: deploy latest version to prod

### PR DESCRIPTION
**Pull request by CalvinRodo**
Deploy the latest server and appstore ECS tasks into production

### CODE REVIEW

File: env/production/ecs/terragrunt.hcl [hunk 1]
```diff
@@ -41,18 +45,31 @@ dependency "s3" {
...
+  billing_tag_key                     = "CostCentre"
+  billing_tag_value                   = "CovidShield"
+  masked_image_tag                    = "7e972df01ac3e1e4226cb91184554d1543b4bff2"
+  unmasked_image_tag                  = "7e972df01ac3e1e4226cb91184554d1543b4bff2"
+  masked_server_tag                   = "7e972df01ac3e1e4226cb91184554d1543b4bff2"
+  unmasked_server_tag                 = "7e972df01ac3e1e4226cb91184554d1543b4bff2"
+  masked_appstore_tag                 = "7e972df01ac3e1e4226cb91184554d1543b4bff2"
+  unmasked_appstore_tag               = "7e972df01ac3e1e4226cb91184554d1543b4bff2"
+  cpu_units                           = 512
+  memory                              = 1024
```
Comment (patheard) [inline env/production/ecs/terragrunt.hcl:69]: Since this is going prod, want to bump this to `4096` or `8192`?

**Review by patheard — CHANGES_REQUESTED** [review]
Comment about ECS task memory.

File: env/production/ecs/terragrunt.hcl (hunk 1 above)
Comment (CalvinRodo) [reply env/production/ecs/terragrunt.hcl:69]: yes should be 8192
also might want to upgrade cpu units as well

Comment (patheard) [reply env/production/ecs/terragrunt.hcl:69]: makes sense for cpu bump as well

**Review by patheard — APPROVED** [review]
`+1`

**Pull request merged**